### PR TITLE
Add cloud storage functionality to portfolio

### PR DESCRIPTION
**Pull request by efrainmunoz100**
- Updates index.html to include a user submission form for images, and a button to see user submissions
- Creates uploadServlet.java to upload user images to cloud storage
- Creates userUploadsServlet.java to show user submissions

********************************************************
Screen Shot 2021-03-09 at 10 29 03 PM
********************************************************
Screen Shot 2021-03-09 at 10 31 21 PM
********************************************************
Screen Shot 2021-03-09 at 10 41 02 PM

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/userUploadsServlet.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        // Add option to return to main page
+        response.getWriter().println("<h1>User Submissions</h1>");
+        response.getWriter().println("<h2>Click <a href=\"/\">here<a/> to return to the main page</h2>");
+
+        // Output <img> elements as HTML.
+        response.setContentType("text/html;");
+        for (Blob blob : blobs.iterateAll()) {
+        String imgTag = String.format("<img src=\"%s\" />", blob.getMediaLink());
+        response.getWriter().println(imgTag);
+        response.getWriter().println("<br>");
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/userUploadsServlet.java:35]: Indent contents of for loop?

**Comment by efrainmunoz100** [comment]
- changed servlet and class names to be in CamelCase
- moved project name declarations out from the function, and declared them as class constants
- changed URLs to follow the hyphenation convention instead of camelCase

File: portfolio/src/main/java/com/google/sps/servlets/FetchServlet.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 /** Responds with text when a request is sent to the /fetch URL */
 @WebServlet ("/fetchServlet")
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/FetchServlet.java:10]: I think this URL and the one in JsonServlet should also be hyphenated, but I'll leave that for a future PR should you choose to address it

File: portfolio/src/main/java/com/google/sps/servlets/UploadServlet.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+/** 
+ * Takes an image uploaded by a user, and uploads it to Google Cloud Storage, then shows the user their uploaded
+ * image.
+ */
+@WebServlet("/upload")
+@MultipartConfig
+public class UploadServlet extends HttpServlet{
+
+    private static final String projectId = "emunoz-sps-spring21";
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/UploadServlet.java:27]: Constants are typically all uppercase and words are separated with underscores, like PROJECT_ID and BUCKET_NAME

**Review by renguyen — APPROVED** [review]
Thanks for cleaning up the code! Some more minor comments about constant names and the other servlets' URLs, but I'll approve for now and you can address them in the future if you have time!

**Pull request merged**